### PR TITLE
Improve Cassandra requirements

### PR DESCRIPTION
**Pull request by mosabua**
I am not sure about all the ports. Who would know more details for this connector? @findinpath @hashhar  @martint ?

### CODE REVIEW

File: docs/src/main/sphinx/connector/cassandra.rst [hunk 1]
```diff
@@ -5,10 +5,18 @@ Cassandra connector
...
--------------
-
-Connector is tested against Cassandra version 2.2 and 3.11, but any intermediate or newer versions are expected to work.
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Cassandra
+data source are:
+
+* Cassandra version 2.2 or higher or PostgreSQL 9.6 or higher
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/cassandra.rst:14]: Why do we need to mention PostgreSQL here?

File: docs/src/main/sphinx/connector/cassandra.rst [hunk 2]
```diff
@@ -5,10 +5,18 @@ Cassandra connector
...
+* Network access from the Trino coordinator and
+  workers to Cassandra:
+  * For cluster communication, default on port 7000
+  * For native protocol client access, default on port 9042
+  * For JMX, default to port 7199
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/cassandra.rst:19]: I think we should document only 9042 port.

Comment (mosabua) [reply docs/src/main/sphinx/connector/cassandra.rst]: yeah .. I wasnt sure. Thanks. Fixed now.

File: docs/src/main/sphinx/connector/cassandra.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/cassandra.rst]: Cut and paste error.. removed.

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**